.tekton/integration-tests/pipelines/tempo-operator-e2e-test-pipeline-4-14.yaml

```diff
@@ -286,6 +286,153 @@ spec:
               check_operator_installed "servicemeshoperator" openshift-operators
               check_operator_installed "serverless-operator" openshift-serverless
               echo "Operator installation check completed."
+    - name: check-tempo-version
+      description: The task checks Tempo operator and operand version details and image info
+      runAfter:
+        - tempo-install
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: namespace
+          value: "$(params.namespace)"
+      taskSpec:
+        params:
+          - name: SNAPSHOT
+          - name: namespace
+            type: string
+        volumes:
+          - name: credentials
+            emptyDir: {}
+        steps:
+          - name: get-kubeconfig
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/build-definitions.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml
+            params:
+              - name: eaasSpaceSecretRef
+                value: $(tasks.eaas-provision-space.results.secretRef)
+              - name: clusterName
+                value: "$(tasks.provision-cluster.results.clusterName)"
+              - name: credentials
+                value: credentials
+          - name: check-version
+            env:
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+            image: registry.redhat.io/openshift4/ose-cli:latest
+            script: |
+              #!/bin/bash
+              set -eu
+
+              # Variables for strings to check
+              JAEGER_QUERY_VERSION="gitVersion\":\"v1.62.0"
+              OPERATOR_VERSION="operator-version\":\"0.14.2"
```

Comment:
If the versions are hardcoded here, we need to update this file for every release?
I'd suggest to use a regular expression to verify if any valid version number is printed

Reply:
Checking if we can pass the values as parameters to the pipeline instead of hardcoding it or using regex.

Reply:
@andreasgerstmayr I updated the task to use parameters passed to the pipeline as vars so its not hardcoded anymore and the task is passing :)

Reply:
I'll add a similar test to the OTEL operator test pipeline as well so that we do not have to verify this info manually.